clarity_ext/domain/result_file.py

```diff
@@ -1,35 +1,35 @@
-from clarity_ext.unit_conversion import UnitConversion
-from clarity_ext.domain.artifact import Artifact
+from clarity_ext.domain.aliquot import Aliquot, Sample
 
 
-class ResultFile(Artifact):
+class ResultFile(Aliquot):
```

Comment:
A result_file in clarity, (not a shared result file) is a rather poor name for it. When it's labelled as 'PerInput' it's about a sample measurement, that has a result file attached to it. Even if Analyte and Result file may have a rather different set of UDFs attached to it, they have sample placement in common. Regarding concentration and volume, they are a very basic measures that also is common to the two classes. 

I'm not convinced that it would be the right thing to cancel the Aliquot class. And certainly not now. 

UDF may be attached to every entity in Clarity, not only artifacts.